README.md

```diff
@@ -35,15 +35,40 @@ FROM carplang/carp:latest
 ### As a carp builder:
 
 ``` bash
-# from this repo root:
+### from this repo root:
 docker run -v $(pwd)/example:/mnt/app \
            --user 1000:1000 \
            --rm \
            carplang/carp:latest \
            carp -b hello_world.carp
 
-# You can run resulting executable from the host(If you run x64 glibc Linux)
+### You can run resulting executable from the host(If it is x64 glibc Linux)
 ./example/out/HelloWorld
+
+### Or you can run your app with a browser(WASM/ASM.JS):
+
+## Generate web files:
+docker run -v $(pwd)/example/out/:/mnt/app \
+            --rm \
+            -ti \
+            carplang/carp:emcc \
+            emcc main.c \
+                 -s WASM=1 \
+                 -I/opt/carp/core \
+                 -o hello.html \
+                 --emrun
+
+## Serve:
+docker run -v $(pwd)/example/out/:/mnt/app \
+           --rm \
+           -ti \
+           --net=host \
```

Comment:
This makes container run on hosts network (so it can be accessed via http://localhost:8080/hello.html)